.gitlab-ci.yml

```diff
@@ -97,6 +96,8 @@ setup_fusillade:
 
 deploy:
   stage: deploy
+  before_script:
```

Comment:
Might be worthwhile to move: `export COMMITS_URL=${GITHUB_API}/repos/HumanCellAtlas/data-store/commits` here too since it's only called for this function.